backend/app/api/community.py

```
@@ -0,0 +1,830 @@
+    # Build query filter - exclude deleted posts AND featured posts
+    query = {"isDeleted": False, "_id": {"$nin": featured_ids}}
+
+    # Add post type filter if specified
+    if postType:
+        query["postType"] = postType
+
+    # Cursor-based pagination - fetch posts before the cursor
+    if cursor:
+        try:
+            query["_id"] = {"$lt": ObjectId(cursor)}
+        except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)